### ISSUE TITLE
train_loss > val_loss

### ISSUE THREAD

**Issue by lucas2606-rs** [ShuGuoJ/3DVSCNN#1]
have you encountered this question during training that train loss is greater than validation loss?
it s strange.................

**Comment by ShuGuoJ** [comment]
That is very abnormal. I suggest you should check your dataset and codes again.
